Tests/StoreKitUnitTests/TestHelpers/StoreKitConfigTestCase+Extensions.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  StoreKitConfigTestCase+Extensions.swift
+//
+//  Created by Nacho Soto on 11/17/22.
+
+import Nimble
+@testable import RevenueCat
+@preconcurrency import StoreKit
+import XCTest
+
+@available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+extension StoreKitConfigTestCase {
```

Comment:
I moved this from `StoreKitTestHelpers.swift` into this file since this is only for `StoreKitConfigTestCase` and not relevant for integration tests.

Reply:
Done.